sys/log/full/src/log.c

```diff
@@ -571,6 +691,10 @@ log_append_prepare(struct log *log, uint8_t module, uint8_t level,
     }
 #endif
 
+#if MYNEWT_VAL(LOG_FLAGS_TRAILER_SUPPORT)
+    ue->ue_flags |= LOG_FLAGS_TRAILER_SUPPORT;
```

Comment:
Not sure what is the purpose of this flag. I assume this should be an indication whether trailer is appended to log entry or not, but it's set for each entry even in case application decides not to append a trailer. How can application figure out if an entry has a trailer when log is read?

Reply:
That's a good catch. Thank you. I will check the existence of `log->l_th` to set the flag.

Reply:
Fixed it.